<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWidget</class>
 <widget class="QWidget" name="ReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="revenueTab">
      <attribute name="title">
       <string>Thống kê Doanh thu</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Từ ngày:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="fromDateEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Đến ngày:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="toDateEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewReportButton">
            <property name="text">
             <string>Xem Báo cáo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="revenueTable">
         <column>
          <property name="text">
           <string>Mã Phiếu Xuất</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ngày Xuất</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tên Khách Hàng</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tổng Tiền</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalRevenueLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tổng Doanh thu: 0 VND</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="inventoryTab">
      <attribute name="title">
       <string>Báo cáo Tồn kho</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="inventoryTable">
         <column>
          <property name="text">
           <string>Mã HH</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tên Hàng Hóa</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Số Lượng Tồn</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
